src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import duke.command.CommandParser;
+import duke.exception.DukeException;
+import duke.storage.FileStorage;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.CommandLineUi;
+import duke.ui.Ui;
+
+import java.util.Scanner;
+
+public class Duke {
```

Comment:
Shouldn't there be a Javadoc comment for public classes?

Reply:
Good catch! Thank you 😄